sysutils/salt/Portfile

```diff
@@ -3,7 +3,7 @@
 PortSystem        1.0
 
 name               salt
-version            2019.2.0
+version            3000.3
```

Comment:
Have upstream changed their version numbering scheme? The old one looks like a date, whereas the new one obviously cannot be ?

Reply:
Yeah. That happened. They are also remarketing Salt as a security platform. Maybe that has something to do with it? I think it got them some black hat attention WRT the CVE...